### DISCUSSION TITLE
Design Ideas

### DISCUSSION THREAD

**Discussion by Alex9THWAVE** [ardura/Actuate#26] [Ideas]
Hi, Ardura! 

I saw your synthesizer and decided to give it a fresher look. I added some new visual representations of functions in some places, while in others, I kept it as it is. 

I couldn't find your contact, email, so I wrote here.

<img width="978" alt="Actuate 2 0 - 01" src="https://github.com/ardura/Actuate/assets/162492825/d383d79e-d4c6-4ef2-a5a3-9ca53966fe02">
<img width="978" alt="Actuate 2 0 - 02" src="https://github.com/ardura/Actuate/assets/162492825/dde7dd19-fff5-4916-95ec-c72ac1cdde59">

**Comment by ardura** [comment]
Wow that looks amazing 😍
I'll admit I'm not a graphics/GUI person as my day job is unrelated to GUI programming, but I really like the style and design you made!

I especially like the filter section. What did you make this in btw? It's a great mockup.

I'll see what I can do to work towards that 😉
